pipelines/rj_cor__precipitacao_alertario_sftp/utils.py

```
@@ -0,0 +1,277 @@
+    ao nome para rastreamento.
+
+    :param dfr: DataFrame a ser salvo.
+    :param data_type: Tipo de dado ("pluviometric" ou "meteorological").
+    :param partition_column: Coluna usada para particionamento.
+    :returns: Caminho do diretório raiz das partições.
+    """
+    if dfr.empty:
+        logger.warning(
+            "DataFrame %s vazio, criando diretório vazio apenas", data_type
+        )
+        base_path = Path(f"/tmp/precipitacao_alertario/{data_type}")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/precipitacao_alertario/"

(S108)